tar/bsdtar.c

```diff
@@ -74,20 +74,13 @@ __FBSDID("$FreeBSD: src/usr.bin/tar/bsdtar.c,v 1.93 2008/11/08 04:43:24 kientzle
  * Per POSIX.1-1988, tar defaults to reading/writing archives to/from
  * the default tape device for the system.  Pick something reasonable here.
  */
-#ifdef __linux
+#if defined(__linux)
 #define	_PATH_DEFTAPE "/dev/st0"
-#endif
-#if defined(_WIN32) && !defined(__CYGWIN__)
+#elif defined(_WIN32) && !defined(__CYGWIN__)
 #define	_PATH_DEFTAPE "\\\\.\\tape0"
```

Comment:
Let's keep cases here for Apple and FreeBSD just to be especially clear:
```
#elif defined(__APPLE__)
#undef _PATH_DEFTAPE  /* Don't use a default tape device on macOS */
#elif defined(__FreeBSD__)
#define _PATH_DEFTAPE "/dev/tape"
```

Reply:
It will be much better, thanks for pointing at it.